JobWorkers/BaseJobWorker.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace JobWorkers;
+
+/**
+ * All user-defined workers indirectly extend this class. Not directly! Use one of:
+ * @see JobWorkerSimple
+ * @see JobWorkerManualRespond
+ * @see JobWorkerNoReply
+ */
+abstract class BaseJobWorker implements \KphpJobWorkerRequest {
+
+  /** @var mixed[] */
+  protected $_untyped_context = [];
+
+  /**
+   * @param mixed[] $untyped_context
+   */
+  protected function saveGlobalsContext(array &$untyped_context) {
+    // to be overridden
+  }
+
+  /**
+   * @param mixed[] $untyped_context
+   */
+  protected function restoreGlobalsContext(array $untyped_context) {
+    // to be overridden
+  }
```

Comment:
Can we add some examples of using these functions?

Reply:
They will be fully covered in a documentation.
This repo is not a documentation. It's just a skeleton for some real project to be used in a copy-paste manner.